common/component_interface_utils/include/component_interface_utils/rclcpp/create_interface.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COMPONENT_INTERFACE_UTILS__RCLCPP__CREATE_INTERFACE_HPP_
+#define COMPONENT_INTERFACE_UTILS__RCLCPP__CREATE_INTERFACE_HPP_
+
+#include <component_interface_utils/rclcpp/service_server.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <utility>
+
+namespace component_interface_utils
+{
+
+/// Create a service wrapper for logging. This is a private implementation.
+template <class SpecT, class NodeT, class CallbackT>
+typename Service<SpecT>::SharedPtr create_service_impl(
+  NodeT * node, CallbackT && callback, rclcpp::CallbackGroup::SharedPtr group = nullptr)
+{
+  // Use a node pointer because shared_from_this cannot be used in constructor.
+  // This function is a wrapper for the following.
+  // https://github.com/ros2/rclcpp/blob/galactic/rclcpp/include/rclcpp/node.hpp#L267-L281
```

Comment:
You should make this a parmanent link.

```suggestion
  // https://github.com/ros2/rclcpp/blob/48068130edbb43cdd61076dc1851672ff1a80408/rclcpp/include/rclcpp/node.hpp#L267-L281
```